runtime/src/bank.rs

```diff
@@ -284,7 +284,7 @@ impl RentDebits {
 }
 
 pub type BankStatusCache = StatusCache<Result<()>>;
-#[frozen_abi(digest = "HEJXoycXvGT2pwMuKcUKzzbeemnqbfrUC4jHZx1ncaWv")]
+#[frozen_abi(digest = "7FSSacrCi7vf2QZFm3Ui9JqTii4U6h1XWYD3LKSuVwV8")]
```

Comment:
Yup, it wouldn't be in there because the `cap_accounts_data_len` feature was never activated. It shouldn't matter anyways if it was because changing the name of the enum wouldn't break serialization.